### PR TITLE
Add support for JS/TS testing

### PR DESCRIPTION
**Pull request by janklimo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ Get started on your new app within minutes instead of hours 🚗💨
     - [React](#react)
     - [TypeScript](#typescript)
     - [Hot Module Replacement (HMR)](#hot-module-replacement-hmr)
+    - [JavaScript/TypeScript tests](#javascripttypescript-tests)
```
Comment (hiattp) [inline README.md]: I know we haven't done this everywhere but I'm in favor of capitalizing section headers (Tests)

File: README.md [hunk 2]
```diff
@@ -92,8 +93,8 @@ All of the following have been installed and pre-configured:
 
 ### Base system
 
-- Rails 5.2.1
-- Ruby 2.5.1
+- Rails 6
+- Ruby 2.6.5
```
Comment (hiattp) [inline README.md]: Should we just say Rails and Ruby instead of trying to keep this updated with version numbers?

**Comment by hiattp** [comment]
🤩

File: README.md (hunk 2 above)
Comment (janklimo) [reply README.md]: Agreed, that info lives in other files. That's actually the reason I wrote Rails 6, not Rails 6.0.2 😄

**Pull request merged**